src/atopile/cli/excepthook.py

```diff
@@ -0,0 +1,103 @@
+import contextlib
+import sys
+from types import ModuleType
+
+import rich
+
+from atopile import telemetry
+from atopile.cli.logging import logger
+from atopile.errors import UserPythonModuleError, _BaseBaseUserException
+
+
+def in_debug_session() -> ModuleType | None:
```

Comment:
I assume there's a good reason this isn't just `faebryk.exceptions...`, but tldr; because it's importing other things we don't want yet

I might advocate for a lazy import instead